galleon-pack/src/main/resources/feature_groups/domain-host-excludes.xml

```diff
@@ -7,34 +7,30 @@
+        <param name="excluded-extensions" value="[&quot;org.wildfly.extension.core-management&quot;,&quot;org.wildfly.extension.datasources-agroal&quot;,&quot;org.wildfly.extension.discovery&quot;,&quot;org.wildfly.extension.ee-security&quot;,&quot;org.wildfly.extension.elytron&quot;,&quot;org.wildfly.extension.microprofile.config-smallrye&quot;,&quot;org.wildfly.extension.microprofile.health-smallrye&quot;,&quot;org.wildfly.extension.microprofile.opentracing-smallrye&quot;,&quot;org.wildfly.extension.microprofile.metrics-smallrye&quot;,&quot;org.wildfly.extension.clustering.web&quot;]"/>
     </feature>
     <feature spec="domain.host-exclude">
         <param name="host-exclude" value="WildFly10.1"/>
-        <param name="excluded-extensions" value="[&quot;org.wildfly.extension.core-management&quot;,&quot;org.wildfly.extension.datasources-agroal&quot;,&quot;org.wildfly.extension.discovery&quot;,&quot;org.wildfly.extension.ee-security&quot;,&quot;org.wildfly.extension.elytron&quot;,&quot;org.wildfly.extension.microprofile.config-smallrye&quot;,&quot;org.wildfly.extension.microprofile.health-smallrye&quot;,&quot;org.wildfly.extension.microprofile.opentracing-smallrye&quot;,&quot;org.wildfly.extension.microprofile.metrics-smallrye&quot;]"/>
+        <param name="excluded-extensions" value="[&quot;org.wildfly.extension.core-management&quot;,&quot;org.wildfly.extension.datasources-agroal&quot;,&quot;org.wildfly.extension.discovery&quot;,&quot;org.wildfly.extension.ee-security&quot;,&quot;org.wildfly.extension.elytron&quot;,&quot;org.wildfly.extension.microprofile.config-smallrye&quot;,&quot;org.wildfly.extension.microprofile.health-smallrye&quot;,&quot;org.wildfly.extension.microprofile.opentracing-smallrye&quot;,&quot;org.wildfly.extension.microprofile.metrics-smallrye&quot;,&quot;org.wildfly.extension.clustering.web&quot;]"/>
     </feature>
     <feature spec="domain.host-exclude">
         <param name="host-exclude" value="WildFly11.0"/>
         <param name="host-release" value="WildFly11.0"/>
-        <param name="excluded-extensions" value="[&quot;org.wildfly.extension.datasources-agroal&quot;,&quot;org.wildfly.extension.ee-security&quot;,&quot;org.wildfly.extension.microprofile.config-smallrye&quot;,&quot;org.wildfly.extension.microprofile.health-smallrye&quot;,&quot;org.wildfly.extension.microprofile.opentracing-smallrye&quot;,&quot;org.wildfly.extension.microprofile.metrics-smallrye&quot;]"/>
+        <param name="excluded-extensions" value="[&quot;org.wildfly.extension.datasources-agroal&quot;,&quot;org.wildfly.extension.ee-security&quot;,&quot;org.wildfly.extension.microprofile.config-smallrye&quot;,&quot;org.wildfly.extension.microprofile.health-smallrye&quot;,&quot;org.wildfly.extension.microprofile.opentracing-smallrye&quot;,&quot;org.wildfly.extension.microprofile.metrics-smallrye&quot;,&quot;org.wildfly.extension.clustering.web&quot;]"/>
```

Comment:
I suggest keeping the alphabetical order here and put org.wildfly.extension.clustering.web exclusion as the first in the list

Reply:
Oh - I didn't realize these were lexicographically ordered.  I figured that adding this subsystem to the end would make reviewing diffs easier.  I'll fix.

Reply:
We are trying to follow an alphabetical order here, but really it's not a requirement, what is useful is to follow the same order than the legacy template: feature-pack/src/main/resources/configuration/domain/template.xml. It is just it helps with comparison between standard and legacy builds.